assets/js/components/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Disclosure, Transition } from '@headlessui/react';
+
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import ProviderSelection from './ProviderSelection';
+
+const ChecksCatalog = () => {
+  const catalog = useSelector((state) => state.catalog.catalog);
+  const providers = catalog.map((provider) => provider.provider);
+  const [selected, setSelected] = useState(providers[0]);
+
+  useEffect(() => {
+    setSelected(providers[0]);
+  }, [providers[0]]);
+
+  return (
+    <div>
+      <ProviderSelection
+        providers={providers}
+        selected={selected}
+        setSelected={setSelected}
```

Comment:
call this prop `onChange` :wink:

Reply:
Thanks!
Done: https://github.com/trento-project/web/pull/241/commits/2204511c4bde44584b72ad4dcd3afbf4d7fcda1d